src/components/MapLegend.vue

```diff
@@ -58,18 +62,64 @@
                 let layer = layers[index];
                 let color = colors[index];
                 let item = document.createElement('div');
+                let keyContainer = document.createElement('div');
+                let textContainer = document.createElement('div');
                 let key = document.createElement('span');
 
+                item.style.display = "flex";
+                item.style.margin = "0 0 5px 0"
+                item.style.padding = "0 10px 0 0";
                 key.style.backgroundColor = color;
-                key.style.marginRight = '5px';
+                key.style.margin = ' 0 5px 0 10px';
                 key.style.display = 'inline-block';
                 key.style.height = '10px';
                 key.style.width = '10px';
                 let value = document.createElement('span');
+                value.style.fontSize = ".8em";
+                value.style.userSelect = "none"
+                let highlight = document.createElement('span');
+                let text = document.createElement('span');
 
-                value.innerHTML = layer;
-                item.appendChild(key);
-                item.appendChild(value);
+                highlight.style.color = color;
+                highlight.style.fontWeight = "bold";
+                highlight.style.textShadow = "1px 1px 0 rgba(0,0,0,.25)"
+
+                let options = {
```

Comment:
This is pretty interesting use of an object and seems like a workable solution. However, it would be good if we could decouple the code from the style. By that I mean, it would be good if when we made changes to the styles, it wouldn't break other parts of the program. 

In this case, since we are basically hard coding the description text, it would be better to do that in the Mapbox styles. We can add an additional key value pair for each of the stops such as
 
 . . .
'showButtonLayerToggle': false,
'very high': ['very high', 'it's typically drier than it is today in this region'],  
'high':[',''] ,
'medium':['Common', 'today is normal for this region'] ,
etc